capra-fagradar/src/tech-leader-radar/organisasjon/sosiotekniske designprinsipper.mdx

```diff
@@ -0,0 +1,21 @@
+---
+id:
+name: Sosiotekniske designprinsipper
+depth: 2
+quadrant: Organisasjon
+is_new: true
+---
+## Hva er det?
+Sosiotekniske designprinsipper er en syntese av forskning og bruk av sosioteknisk systemdesign siden 50-tallet. De oppsummerer
+designpraksiser som har vist seg sentrale når for å optimalisere arbeidsprosesser og systemer ved å kombinerer både sosiale og
```

Comment:
De designpraksisene, det er litt uhåndgripelig for meg. Jeg vet ikke om det gir mening, men kunne vi hatt et eksempel eller forklart Iitt nærmere hva som menes?

Reply:
Det er henvist til det lenger nede. Tenker det ikke er plassen her til på gjennom alle 20.